cmd/influxd/launcher/launcher.go

```diff
@@ -692,13 +701,24 @@ func (m *Launcher) run(ctx context.Context) (err error) {
 		return err
 	}
 
+	// Enable storage layer page fault limiting if rate set above zero.
+	var pageFaultLimiter *rate.Limiter
+	if m.pageFaultRate > 0 {
+		pageFaultLimiter = rate.NewLimiter(rate.Limit(m.pageFaultRate), 1)
```

Comment:
How does the `1` map to the `mmap` instance here? Should this be a `const` so it's more descriptive? Could/should this change ever?

Reply:
It's part of the `golang.org/x/time/rate` package. It's the `burst` value for the limiter, although I couldn't get it to work correctly with any other value than `1`. https://godoc.org/golang.org/x/time/rate#NewLimiter

It seems like it lets in a higher number of values than what's available in the token bucket—i.e. if you have a `burst` of `10` and you pass in `10` immediately (before the token bucket has filled) then it allows it and doesn't wait at all. Instead we just call the underlying `Wait()` several times. It's something we can look at in the future if it seems to be a performance issue but I'm not especially worried about it right now.